### DISCUSSION TITLE
画像ダウンロードボタンの実装について

### DISCUSSION THREAD

**Discussion by kazzyfrog** [lgtm-factory/lgtm-factory#84] [Scrap]
これは、いろんな方法で、実装できそうな気がしてます。。！

どんな実装ができるのか、
気になるので、つらつらメモしていきますー🦀

**Comment by kazzyfrog** [comment]
aタグのダウンロード属性でも、ファイルのダウンロードはできそう：
https://developer.mozilla.org/ja/docs/Web/HTML/Element/a

**Reply by kagomen** [reply]
そんなやり方もあるんですね！シンプルで良さそう！🐢
